ProjectSettings/ProjectVersion.txt

```diff
@@ -1 +1 @@
-m_EditorVersion: 2018.3.3f1
+m_EditorVersion: 2018.3.4f1
```

Comment:
Yes, please downgrade back to 3f1. You can download Unity 2018.3.3 at https://unity3d.com/get-unity/download/archive